shell/client/_topbar.scss

```diff
@@ -411,9 +411,6 @@ body>.topbar {
     >.account, >.login {
       >button.show-popup {
         @extend %hamburger-menu-item;
-        >a.expiring-soon {
```

Comment:
There appears to be no `expiring-soon` in any of our HTML templates, so this is dead and I'm removing it.